### DISCUSSION TITLE
Biotite dssp not running! Subprocess error: code 6

### DISCUSSION THREAD

**Discussion by intomybioverse** [biotite-dev/biotite#606] [Q&A] [answered]
Hello everyone.

I am trying to run the biotite secondary structure analysis program in wsl to study the secondary structure of a protein after md simulation. The website link for the python code is provided below.

[Secondary structure of protein during an MD simulation using Biotite] ([Secondary structure during an MD simulation — Biotite 0.41.0 documentation ](https://www.biotite-python.org/examples/gallery/structure/trajectory_sse.html))

However, I am getting the following error:
Traceback (most recent call last):
File “/home/sobs/AvrPto/6M/trajectory_sse.py”, line 43, in
app.join()
File “/home/sobs/.local/biotite_env/lib/python3.10/site-packages/biotite/application/application.py”, line 63, in wrapper
return func(*args, **kwargs)
File “/home/sobs/.local/biotite_env/lib/python3.10/site-packages/biotite/application/localapp.py”, line 257, in join
self.evaluate()
File “/home/sobs/.local/biotite_env/lib/python3.10/site-packages/biotite/application/dssp/app.py”, line 90, in evaluate
super().evaluate()
File “/home/sobs/.local/biotite_env/lib/python3.10/site-packages/biotite/application/localapp.py”, line 278, in evaluate
raise SubprocessError(
subprocess.SubprocessError: ‘mkdssp’ returned with exit code -6: mkdssp: ./src/CifParser.cpp:330: cif::SacParser::CIFToken cif::SacParser::getNextToken(): Assertion `mTokenValue.length() >= 3’ failed.

I am using python3.10 for this program, biotite has been installed, it has a sub-package dssp. Additionally, I have installed dssp as well. Yet, I am getting this error. Can you please help out?

**Comment by padix-key** [comment, marked as answer]
Thanks for the report. I moved it into an issue: #607
